ansible/vars.yml

```diff
@@ -15,8 +15,8 @@ postgrest_release: "12.0.2"
 postgrest_arm_release_checksum: sha1:a08eaa2af548d44b4c8ea61b0223fb7019f5c768
 postgrest_x86_release_checksum: sha1:40f65ded06b9de6567fbe2cd7a317196e22dd595
 
-gotrue_release: 2.148.0
-gotrue_release_checksum: sha1:e3ab39c10605b6a00a953528022b563c8fa1ca33
+gotrue_release: 2.153.0
```

Comment:
I'd prefer upgrade of extensions be in a separate PR

Reply:
@pashkinelfe I needed to test these upgraded versions in staging. Although typically I would agree with you.

Reply:
resolved with https://github.com/supabase/postgres/pull/953/commits/e72391df5917df169a3e8329a82a416580cbdda0